src/Native/System.Net.Security.Native/pal_gssapi.h

```diff
@@ -39,11 +39,18 @@ enum PAL_GssFlags : uint32_t
     PAL_GSS_C_DELEG_POLICY_FLAG = 0x8000
 };
 
+/*
+Issue: #7342
+Disable padded warning which occurs in case of arm32 builds
+*/
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wpadded"
 struct PAL_GssBuffer
 {
     uint64_t length;
     uint8_t* data;
 };
+#pragma clang diagnostic pop
```

Comment:
This struct needs to be the exact same size and layout of its managed counterpart:
https://github.com/dotnet/corefx/blob/f13855b6669e531a702f40e2306f71e3755ed6c4/src/Common/src/Interop/Unix/System.Net.Security.Native/Interop.GssBuffer.cs#L14-L18
Isn't this warning saying that the size we expect the struct to be isn't what it'll actually be?  That'll cause memory corruption and the like when we try to marshal this.